tasks/selinux.yml

```diff
@@ -14,6 +14,16 @@
     state: present
   when: ansible_distribution == 'Debian' or ansible_distribution == 'Ubuntu'
 
+- name: "authorize {{ ssh_server_ports }} ports for selinux"
+  seport:
+    ports: '{{ item }}'
+    proto: tcp
+    setype: ssh_port_t
+    state: present
+  with_items:
+    - "{{ ssh_server_ports }}"
+  when: ssh_server_ports != ['22']
```

Comment:
Let's remove this when-condition since the seport-module is idemopent and will not change anything here.

Reply:
It’s because I’m too impatient, I’d like to skip tasks if unnecessary. I’m changin' this !